config/app.php

```diff
@@ -67,7 +67,7 @@
     |
     */
 
-    'timezone' => 'UTC',
+    'timezone' => 'Europe/Budapest',
```

Comment:
same

Reply:
Do you think that dates of events happening in Budapest should not be stored and displayed according to local time?